### PR TITLE
WIP: Add script that allows you to run tests against plugins in PRs

### PR DESCRIPTION
**Pull request by JPinkney**
### What does this PR do?

This PR adds in the ability to run language tests against incoming PRs.

It is currently hardcoded to run against a previous PR for testing purposes

### CODE REVIEW

**Comment by JPinkney** [comment]
[ci-test]

File: .ci/functional-tests-utils.sh [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+      echo "Github rate is lower than 1000. *Not* using che-bot for minishift startup."
+      echo "If minishift startup fails, please try again later."
+    fi
+  fi
+
+  minishift version
+  minishift config set memory 14GB
+  minishift config set cpus 4
+
+  echo "======== Lunch minishift ========"
```
Comment (amisevsk) [inline .ci/functional-tests-utils.sh]: *Launch*

:)

Comment (JPinkney) [reply .ci/functional-tests-utils.sh]: Yeah that was a copy from the upstream script haha

**Comment by JPinkney** [comment]
[ci-test]

**Comment by benoitf** [comment]
Hi, do you think it can work on github actions ?
Instead of cico scripts

**Comment by JPinkney** [comment]
@benoitf Probably? I'm not really familiar with github actions. I'm just trying to take this PR far enough so that someone can take this over at this point

**Comment by benoitf** [comment]
@JPinkney well github actions is also just a list of bash commands that you can run on a given node so basically it's quite the same

**Comment by JPinkney** [comment]
[ci-test]

**Comment by benoitf** [comment]
it has been replaced by https://github.com/eclipse/che-plugin-registry/pull/593

**Pull request closed without merging**